content/md/en/docs/reference/transaction-format.md

```diff
@@ -84,24 +84,38 @@ The way applications know how to construct a transaction correctly is provided b
 An application will know how to correctly encode a transaction by using the metadata types and transaction format.
 If a call doesn't need to be signed, then the first bit in `[2]` will be 0 and so an application will know not to try decoding a signature.
 
-**Example:**
+**Polkadot JS Apps example:**
 
-Balances transfer from Bob to Dave: Bob sends `42` units to Dave.
+Here we demonstrate a detailed, manual extrinsic construction an submission of a balance transfer from Bob to Dave: Bob sends `42 UNIT`s to Dave.
 
-[ TODO: polkadotjs apps screenshot ]
+1. Start a [node template](https://github.com/substrate-developer-hub/substrate-node-template) in `--dev` mode (see the [quick start](/quick-start) guide for how to set this up)
+1. Navigate to <https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/extrinsics>
+1. Set `Bob` as sender and select `balances` pallet and it's `transfer(dest, value)` call
```

Comment:
shouldn't this be "...the `transfer(dest, value)` call"?

Reply:
open to a direct suggestion @IkerAlus  - use `ctrl+g` to pull that up in a comment here ;)

Reply:
```suggestion
1. Set `Bob` as sender and select `balances` pallet and the `transfer(dest, value)` call
```